run_linux.go

```diff
@@ -1736,6 +1738,13 @@ func (b *Builder) runSetupVolumeMounts(mountLabel string, volumeMounts []string,
 			}
 			return overlayMount, err
 		}
+		if foundU {
+			if err := filepath.Walk(host, func(filePath string, f os.FileInfo, err error) error {
+				return os.Lchown(filePath, rootUID, rootGID)
```

Comment:
Would we want this UID and GID to respect the --user flag of the container?

For example, if I am in a rootless container and I execute a command like

podman run -ti --user mariadb --userns=keep-id ~/data:/var/lib/data:ZU -d mariadb 

I would want the ~/data directory labeled with the mariadb user inside of the container.  This is a very difficult situation for the user to figure out what the correct UID for the data directory would be.

Reply:
In podman that's how it will work. There's an execUser variable that holds the correct UID and GID

```
./bin/podman run -d -e MYSQL_ROOT_PASSWORD=root --user mysql --userns=keep-id -v ~/data:/var/lib/mysql:Z,U --name foo mysql

stat -c "%u:%g" ~/data
100999:100999
```

In the case of buildah, I am making a mistake since I'm using rootUID, rootGID. I'll work on getting the correct UID,GID if --user is specified. Thanks